key/src/main/java/software/amazon/kms/key/ReadHandler.java

```diff
@@ -28,10 +36,7 @@
             .then(
                 progress -> proxy.initiate("kms::describe-key", proxyClient, model, callbackContext)
                     .translateToServiceRequest(Translator::describeKeyRequest)
-                    .makeServiceCall((describeKeyRequest, proxyInvocation) -> proxyInvocation
-                        .injectCredentialsAndInvokeV2(describeKeyRequest,
-                            proxyInvocation.client()::describeKey))
-                    .handleError(BaseHandlerStd::handleNotFound)
```

Comment:
For my own clarity:
This used to result in `ProgressEvent.defaultFailureHandler(e, HandlerErrorCode.NotFound)`, but now we just let `CfnNotFoundException` be thrown. Are those equivalent?

Reply:
Yes, the CloudFormation handler wrapper catches all of these Cfn exceptions and returns the progress event with the corresponding error code. See the wrapper code [here](https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/blob/544bb2c22534b81bc7a03e01f2566dcbc0c63e3a/python/rpdk/java/templates/generate/HandlerWrapper.java#L97-L120).